src/main/java/com/team3/central/mappers/EventMapper.java

```diff
@@ -61,7 +62,7 @@ public EventWithPlaces convertToEventWithPlaces(
       place.setFree(reservation.getReservationToken()
           == null); // If reservation token is null, then place is free
       return place;
-    }).collect(Collectors.toList()));
+    }).sorted(Comparator.comparing(Place::getId)).collect(Collectors.toList()));
```

Comment:
Does BE have to do it? Why not at FE and mobile ?

Reply:
Aleksy asked whether we sort it or not. So I decided that we can sort it. IDK how other teams are doing it (czy they sort it in BE or FE), so I think it won't hurt to do it on BE